src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/Coverage/GroupCoverageTests.kt

```diff
@@ -99,15 +148,152 @@ class GroupCoverageTests : CoverageTests()
+
+        val testYear = 1980
+        val testTarget = BigDecimal(1000)
+        val testCoverage = "0.9".toDecimalOrNull()!!
+
+        JooqContext().use {
+            addCoverageData(it, touchstoneStatus = "open", testYear = testYear,
+                    target = testTarget, coverage = testCoverage, uniformData=true)
+            userHelper.setupTestUser(it)
+        }
+
+        val response = requestHelper.get("$url", minimumPermissions, acceptsContentType = "text/csv")
+
+        val csv = StringReader(response.text)
+                .use { CSVReader(it).readAll() }
+
+        val headers = csv.first().toList()
+        val expectedHeaders = listOf("scenario", "set_name", "vaccine", "gavi_support", "activity_type",
+                "country_code", "country", "year", "age_first", "age_last", "age_range_verbatim", "target",
+                "coverage")
+        headers.forEachIndexed { index, h ->
+            Assertions.assertThat(h).isEqualTo(expectedHeaders[index])
+        }
+
+        val firstRow = csv.drop(1).first().toList()
+        val expectedTarget = "1000".toBigDecimalOrNull()
+        val expectedCoverage = "0.9".toBigDecimalOrNull()
+
+        //test all target values
+        Assertions.assertThat(firstRow[11].toBigDecimalOrNull()).isEqualTo(expectedTarget)
+        Assertions.assertThat(firstRow[12].toBigDecimalOrNull()).isEqualTo(expectedCoverage)
+    }
+
+    @Test
+    fun `wide coverage data for responsibility rows have expected target and coverage values`()
+    {
+        val userHelper = TestUserHelper()
+        val requestHelper = RequestHelper()
+
+        val testYear = 1980
+        val testTarget = BigDecimal(1000)
+        val testCoverage = "0.9".toDecimalOrNull()!!
+
+        JooqContext().use {
+            addCoverageData(it, touchstoneStatus = "open", testYear = testYear,
+                    target = testTarget, coverage = testCoverage, uniformData=true)
+            userHelper.setupTestUser(it)
+        }
+
+        val response = requestHelper.get("$url?format=wide", minimumPermissions, acceptsContentType = "text/csv")
+
+        val csv = StringReader(response.text)
+                .use { CSVReader(it).readAll() }
+
+        //Headers:
+        //0: "scenario", 1: "set_name", 2: "vaccine", 3: "gavi_support", 4: "activity_type",
+        //5: "country_code", 6: "country", 7: "age_first", 8: "age_last", 9: "age_range_verbatim", 10: "coverage_$testYear",
+        //11: "coverage_1985", 12: "coverage_1990", 13: "coverage_1995", 14: "coverage_2000",
+        //15: "target_$testYear", 16: "target_1985", 17: "target_1990", 18: "target_1995", 19: "target_2000"
+
+        val firstRow = csv.drop(1).first().toList()
+        val expectedAggregatedTarget = "1000".toBigDecimalOrNull()
+        val expectedAggregatedCoverage = "0.9".toBigDecimalOrNull()
+
+        //test all target values
+        Assertions.assertThat(firstRow[15].toBigDecimalOrNull()).isEqualTo(expectedAggregatedTarget)
```

Comment:
In this context would it make a bit more sense to just check the actual string value against what we expect, rather than casting them both using this 'BigDecimalOrNull' method? So check that e.g. `firstRow[15] == "100"` with no zeros as we expect? I think that's what we're really trying to test here. Similarly for these other tests...